.github/workflows/validation.yml

```
@@ -9,4 +9,4 @@ on:
 
 jobs:
   validation:
-    uses: JeffersonLab/clas12-validation/.github/workflows/ci.yml@main
+    uses: JeffersonLab/clas12-validation/.github/workflows/ci.yml@java17
```

Revert this, now that https://github.com/JeffersonLab/clas12-validation/pull/87 is merged (and the `java17` branch is deleted)
```suggestion
    uses: JeffersonLab/clas12-validation/.github/workflows/ci.yml@main
```

---

resolved by #191